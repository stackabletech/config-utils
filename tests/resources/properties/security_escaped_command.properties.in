# We must also recognize commands where the colon was escaped
test.from.env=${env\:ENV_TEST}
test.from.file=${file\:UTF-8\:${env\:${env\:FILE_TEST_42_FILE_ENV_NAME}}}
