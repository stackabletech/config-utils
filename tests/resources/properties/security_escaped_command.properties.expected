# We must also recognize commands where the colon was escaped
test.from.env=foo
test.from.file=42
